code/game/gamemodes/nuclear/nuclear_challenge.dm

```diff
@@ -10,9 +10,9 @@
 	icon = 'icons/obj/device.dmi'
 	icon_state = "gangtool-red"
 	item_state = "walkietalkie"
-	desc = "Use to send a declaration of hostilities to the target, delaying your shuttle departure for 20 minutes while they prepare for your assault.  \
-	Such a brazen move will attract the attention of powerful benefactors within the Syndicate, who will supply your team with a massive amount of bonus telecrystals.  \
-	Must be used within five minutes, or your benefactors will lose interest."
+	desc = "Use to send a declaration of hostilities to the target, delaying your shuttle departure for 20 minutes while they prepare for your assault. \
+	Such a brazen move will attract the attention of powerful benefactors within the Syndicate, who will supply your team with a massive amount of bonus telecrystals. \
+	Must be used within ten minutes, or your benefactors will lose interest."
```

Comment:
Not sure if this is true, im pretty sure its dumb, but it includes the time in the lobby too. If you want this to be accurate, its around 6 minutes I believe, depending on lobby time

Reply:
https://github.com/ParadiseSS13/Paradise/pull/9112